src/client/ui/EditorContainer.js

```diff
@@ -86,6 +87,8 @@ class EditorContainer extends Component {
           }
         }
       },
+      DialogComponent: null,
+      dialogProps: {},
       openModal: null,
```

Comment:
I think you can kill all the `openModal` stuff now.

Reply:
I think we're going to use that modal system for the export modal. We can move to using a context based API though.